python/openchange/provision.py

```diff
@@ -621,12 +638,16 @@ def newuser(names, lp, creds, username=None, mail=None):
 add: legacyExchangeDN
 legacyExchangeDN: /o=%(firstorg)s/ou=%(firstou)s/cn=Recipients/cn=%(username)s
 add: proxyAddresses
+proxyAddresses: SMTP:%(smtp_user)s
```

Comment:
In some edge cases the specification uses the proxyAddresses order to choose what of them will use.
This is not our case with NSPI, because this situation never arises but I like more this order, given priority to the smtp addresses and lowest priority to the sysadmin address.